accounts/usbwallet/ledger.go

```diff
@@ -146,6 +174,12 @@ func (w *ledgerDriver) Heartbeat() error {
 // Derive implements usbwallet.driver, sending a derivation request to the Ledger
 // and returning the Ethereum address located on that derivation path.
 func (w *ledgerDriver) Derive(path accounts.DerivationPath) (common.Address, error) {
+	if w.app == ledgerBLSsigner {
+		return common.HexToAddress("0x0000000000000000000000000000000000000001"), nil
```

Comment:
this appears in a few places. how about making it a constant to be used by all of these?

Reply:
Done